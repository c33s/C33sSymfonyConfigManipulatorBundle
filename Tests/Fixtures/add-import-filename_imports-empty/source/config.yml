# Please do not edit this file manually. Manual edits will be lost during construction-kit related commands.
imports: ~
