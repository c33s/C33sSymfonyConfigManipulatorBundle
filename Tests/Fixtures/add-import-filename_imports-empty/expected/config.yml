# Please do not edit this file manually. Manual edits will be lost during construction-kit related commands.
imports:
    - { resource: config/new_config.yml }
